### ISSUE TITLE
Slate does not round trip for some examples of markdown annotations around variables 

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/markdown-transform#215]
**Describe the bug**
Some examples of markdown annotations (e.g., bold/italics) around CiceroMark variables disappear when doing a round-trip through the Slate Document Object Model.

For instance:
```
bash-3.2$ cat test.md 
cat test.md 
PROMISSORY NOTE
====

**<variable id="amount" value="1%2C000.00%20USD" format="0%2C0.00%20CCC"/>** **<variable id="date" value="01%2F30%2F2018"/>**

FOR VALUE RECEIVED, the undersigned, **<variable id="maker" value="%22Daniel%22"/>**, **<if id="individual" value="an%20individual%20residing" whenTrue="an%20individual%20residing" whenFalse="a%20company%20registered"/>** at **<variable id="makerAddress" value="%221%20Main%20Street%22"/>** (“Maker”), hereby promises to pay to the order of **<variable id="lender" value="%22Clause%20Inc.%22"/>**, a **<variable id="legalEntity" value="CORP"/>**, having offices at **<variable id="lenderAddress" value="%22246%205th%20Ave%2C%203rd%20Fl%2C%20New%20York%2C%20NY%2010001%22"/>**, or its successors and permitted assigns (“Lender” or the “Company”), the principal sum of **<variable id="principal" value="500.00%20USD" format="0%2C0.00%20CCC"/>**, plus any and all interest accrued thereon at the Note Rate (defined below), each due and payable in cash in lawful money of the United States on the dates and in the manner set forth in this Promissory Note (this “Note”).
....
bash-3.2$ markus normalize --sample test.md --output test3.md --slate 
2:09:01 PM - info: Creating file: test3.md
2:09:01 PM - info: 
PROMISSORY NOTE
====

<variable id="amount" value="1%2C000.00%20USD" format="0%2C0.00%20CCC"/> <variable id="date" value="01%2F30%2F2018"/>

FOR VALUE RECEIVED, the undersigned, <variable id="maker" value="%22Daniel%22"/>, <if id="individual" value="an%20individual%20residing" whenTrue="an%20individual%20residing" whenFalse="a%20company%20registered"/> at <variable id="makerAddress" value="%221%20Main%20Street%22"/> (“Maker”), hereby promises to pay to the order of <variable id="lender" value="%22Clause%20Inc.%22"/>, a <variable id="legalEntity" value="CORP"/>, having offices at <variable id="lenderAddress" value="%22246%205th%20Ave%2C%203rd%20Fl%2C%20New%20York%2C%20NY%2010001%22"/>, or its successors and permitted assigns (“Lender” or the “Company”), the principal sum of <variable id="principal" value="500.00%20USD" format="0%2C0.00%20CCC"/>, plus any and all interest accrued thereon at the Note Rate (defined below), each due and payable in cash in lawful money of the United States on the dates and in the manner set forth in this Promissory Note (this “Note”).
....
bash-3.2$ 
```

**Comment by jeromesimeon** [comment]
This bug also affects the current master branch based on Slate 0.57:

```
bash-3.2$ ~/git/markdown-transform/packages/markdown-cli/index.js normalize --sample test.md --output test3.md --slate 
2:15:06 PM - info: Creating file: test3.md
2:15:06 PM - info: 
PROMISSORY NOTE
====

<variable id="amount" value="1%2C000.00%20USD" format="0%2C0.00%20CCC"/> <variable id="date" value="01%2F30%2F2018"/>

FOR VALUE RECEIVED, the undersigned, <variable id="maker" value="%22Daniel%22"/>, <if id="individual" value="an%20individual%20residing" whenTrue="an%20individual%20residing" whenFalse="a%20company%20registered"/> at <variable id="makerAddress" value="%221%20Main%20Street%22"/> (“Maker”), hereby promises to pay to the order of <variable id="lender" value="%22Clause%20Inc.%22"/>, a <variable id="legalEntity" value="CORP"/>, having offices at <variable id="lenderAddress" value="%22246%205th%20Ave%2C%203rd%20Fl%2C%20New%20York%2C%20NY%2010001%22"/>, or its successors and permitted assigns (“Lender” or the “Company”), the principal sum of <variable id="principal" value="500.00%20USD" format="0%2C0.00%20CCC"/>, plus any and all interest accrued thereon at the Note Rate (defined below), each due and payable in cash in lawful money of the United States on the dates and in the manner set forth in this Promissory Note (this “Note”).
bash-3.2$ ~/git/markdown-transform/packages/markdown-cli/index.js normalize --sample test.md --output test3.md --cicero
~/git/markdown-transform/packages/markdown-cli/index.js normalize --sample test.md --output test3.md --cicero
2:15:24 PM - info: Creating file: test3.md
2:15:24 PM - info: 
PROMISSORY NOTE
====

**<variable id="amount" value="1%2C000.00%20USD" format="0%2C0.00%20CCC"/>** **<variable id="date" value="01%2F30%2F2018"/>**

FOR VALUE RECEIVED, the undersigned, **<variable id="maker" value="%22Daniel%22"/>**, **<if id="individual" value="an%20individual%20residing" whenTrue="an%20individual%20residing" whenFalse="a%20company%20registered"/>** at **<variable id="makerAddress" value="%221%20Main%20Street%22"/>** (“Maker”), hereby promises to pay to the order of **<variable id="lender" value="%22Clause%20Inc.%22"/>**, a **<variable id="legalEntity" value="CORP"/>**, having offices at **<variable id="lenderAddress" value="%22246%205th%20Ave%2C%203rd%20Fl%2C%20New%20York%2C%20NY%2010001%22"/>**, or its successors and permitted assigns (“Lender” or the “Company”), the principal sum of **<variable id="principal" value="500.00%20USD" format="0%2C0.00%20CCC"/>**, plus any and all interest accrued thereon at the Note Rate (defined below), each due and payable in cash in lawful money of the United States on the dates and in the manner set forth in this Promissory Note (this “Note”).
```

**Comment by jeromesimeon** [comment]
(in master) In CiceroMark we have a `Strong` node around the variable:
```
{
          "$class": "org.accordproject.commonmark.Strong",
          "nodes": [
            {
              "$class": "org.accordproject.ciceromark.Variable",
              "id": "principal",
              "value": "500.00 USD",
              "format": "0,0.00 CCC"
            }
          ]
        }
```
But this gets pushed into a `bold:true` on the text inside the variable in Slate, not sure if that's right:
```
{
            "object": "inline",
            "type": "variable",
            "data": {
              "id": "principal",
              "format": "0,0.00 CCC"
            },
            "children": [
              {
                "object": "text",
                "text": "500.00 USD",
                "bold": true
              }
            ]
          }
```

**Comment by jeromesimeon** [comment]
Fixed in #216
